OpenRA.Game/WPos.cs

```diff
@@ -65,7 +65,7 @@ public static WPos LerpQuadratic(WPos a, WPos b, WAngle pitch, int mul, int div)
 
 			// Add an additional quadratic variation to height
 			// Attempts to avoid integer overflow by keeping the intermediate variables reasonably sized
-			var offset = (int)(((((((long)(b - a).Length * mul) / div) * (div - mul)) / div) * pitch.Tan()) / 1024);
+			var offset = (int)(((((((decimal)(b - a).Length * mul) / div) * (div - mul)) / div) * pitch.Tan()) / 1024);
```

Comment:
You can now drop most of the nested parenthesis, and the comment above will need to be updated.